src/Symfony/Component/HttpKernel/KernelInterface.php

```diff
@@ -21,6 +21,8 @@
  * It manages an environment made of application kernel and bundles.
  *
  * @author Fabien Potencier <fabien@symfony.com>
+ *
+ * @method string getProjectDir()
```

Comment:
```suggestion
 * @method string getProjectDir() Gets the project dir (path of the project's composer file) - not defining it is deprecated since Symfony 4.2
```

Reply:
fingers crossed `span class="pl-s1">` is a bug on github side

![image](https://user-images.githubusercontent.com/1047696/47077033-5accbc00-d200-11e8-875f-9811883f7548.png)